applications/crossbar/src/api_util.erl

```diff
@@ -44,7 +44,7 @@
 -define(DATA_SCHEMA
         ,wh_json:from_list([{<<"type">>, <<"object">>}
                             ,{<<"description">>, <<"The request data to be processed">>}
-                            ,{<<"required">>, 'true'}
+                            ,{<<"required">>, 'false'}
```

Comment:
why ? if the json payload has a `data` element it will validate and context_req_data has its value, otherwise `context:json_request` will have the submitted payload. cb modules handle * req_data`, others like incoming webhooks don't send the `data` object because they have their own schema.